app/models/data.py

```diff
@@ -61,37 +60,63 @@ def __init__(self, **kwargs):
 class CoreData(db.Model, DataMixin):
     __tablename__ = 'core_data'
 
-    last_update_time = db.Column(db.DateTime(timezone=True), nullable=False)
-    last_check_time = db.Column(db.DateTime(timezone=True), nullable=False)
-
-    # the day we mean to report this data for; meant for "states daily" extraction
-    data_date = db.Column(db.Date, nullable=False)
-
-    tests = db.Column(db.Integer)
-    # TODO: additional cols to follow for positives, negatives, hospitalization data, etc.
+    # composite PK: state_name, batch_id
+    state = db.Column(db.String, db.ForeignKey('states.state'), 
+        nullable=False, primary_key=True)
+    batchId = db.Column(db.Integer, db.ForeignKey('batches.batchId'),
+        nullable=False, primary_key=True)
 
-    checker = db.Column(db.String(100))
-    double_checker = db.Column(db.String(100))
-    public_notes = db.Column(db.String)
+    # data columns
+    positive = db.Column(db.Integer)
+    negative = db.Column(db.Integer)
+    pending = db.Column(db.Integer)
+    hospitalizedCurrently = db.Column(db.Integer)
+    hospitalizedCumulative = db.Column(db.Integer)
+    inIcuCurrently = db.Column(db.Integer)
+    inIcuCumulative = db.Column(db.Integer)
+    onVentilatorCurrently = db.Column(db.Integer)
+    onVentilatorCumulative = db.Column(db.Integer)
+    recovered = db.Column(db.Integer)
+    death = db.Column(db.Integer)
 
     # from worksheet, "Notes" column (made by checker or doublechecker)
-    private_notes = db.Column(db.String)
+    privateNotes = db.Column(db.String)
+    # Public Notes related to state
+    notes = db.Column(db.String)
 
-    # from state matrix: which columns?
-    source_notes = db.Column(db.String)
+    lastUpdateTime = db.Column(db.DateTime(timezone=True), nullable=False)
+    lastCheckTime = db.Column(db.DateTime(timezone=True), nullable=False)
 
-    # composite PK: state_name, batch_id
-    state_name = db.Column(db.String, db.ForeignKey('states.state_name'), 
-        nullable=False, primary_key=True)
-    batch_id = db.Column(db.Integer, db.ForeignKey('batch.batch_id'),
-        nullable=False, primary_key=True)
+    # the day we mean to report this data for; meant for "states daily" extraction
+    dataDate = db.Column(db.Date, nullable=False)
```

Comment:
Elliot wants dataDate to be renamed publicationDate now or soon.

Reply:
actually, in the current API it's just `date`... going to use that for parity for now